chromadb/test/property/strategies.py

```diff
@@ -130,13 +160,23 @@ class Collection:
     known_metadata_keys: Dict[str, st.SearchStrategy]
     known_document_keywords: List[str]
     has_documents: bool = False
-    embedding_function: Optional[Callable[[str], types.Embedding]] = lambda x: []
+    has_embeddings: bool = False
+    embedding_function: Optional[types.EmbeddingFunction] = None
 
 
 @st.composite
-def collections(draw, add_filterable_data=False, with_hnsw_params=False):
+def collections(
+    draw,
+    add_filterable_data=False,
+    with_hnsw_params=False,
+    has_embeddings: Optional[bool] = None,
+    has_documents: Optional[bool] = None,
+) -> Collection:
     """Strategy to generate a Collection object. If add_filterable_data is True, then known_metadata_keys and known_document_keywords will be populated with consistent data."""
 
+    if has_embeddings is not None and has_documents is not None:
```

Comment:
nit: you don't need the if and the assert you can just assert that both are not none